GainStation13/code/modules/clothing/under/jobs/modcivilian.dm

```diff
@@ -93,16 +94,33 @@
 			for(O in U.internal_organs) //check the user for the organs they have
 				if(istype(O, /obj/item/organ/genital/belly)) //if that organ is a belly
 					G = O //treat that organ as a genital
-					if(!adjusted) //check the style, if it needs to be the adjusted variants
-						if(G.size <= 9) //check that the size is within accepted values, NOTE: these need to be removed later, better to cap organ sizes to begin with
-							. += mutable_appearance(icon_location, "belly_[G.size]", GENITALS_UNDER_LAYER) //add, from the clothes' icon file the overlay corresponding to that genital at that size and draw it onto the layer
-						else //if not an expected size value, bigger than the max, default to max size
-							. += mutable_appearance(icon_location, "belly_9", GENITALS_UNDER_LAYER)
-					else //use the alternative adjusted sprites
-						if(G.size <= 9)
-							. += mutable_appearance(icon_location, "belly_[G.size]_d", GENITALS_UNDER_LAYER)
-						else
-							. += mutable_appearance(icon_location, "belly_9_d", GENITALS_UNDER_LAYER)
+					// Change to visually update sprite depending on fullness.
+					if(ishuman(O.owner))
+						var/mob/living/carbon/human/H = O.owner
+						var/size = 0
+						var/used_icon_location = icon_location
+						switch(H.fullness)
+							if(0 to FULLNESS_LEVEL_BLOATED) // Normal
+								size = G.size 
+							if(FULLNESS_LEVEL_BLOATED to FULLNESS_LEVEL_BEEG) // Take the stuffed sprite of the same size
+								size = G.size
+								used_icon_location = icon_stuffed_location
+							if(FULLNESS_LEVEL_BEEG to FULLNESS_LEVEL_NOMOREPLZ) // Take the stuffed sprite of size + 1
+								size = G.size + 1
+								used_icon_location = icon_stuffed_location
+							if(FULLNESS_LEVEL_NOMOREPLZ to INFINITY)// Take the stuffed sprite of size + 2
+								size = G.size + 2
+								used_icon_location = icon_stuffed_location
```

Comment:
```suggestion

						switch(H.fullness)
							if(0 to FULLNESS_LEVEL_BLOATED) // Normal
								size = G.size 
							if(FULLNESS_LEVEL_BLOATED to FULLNESS_LEVEL_BEEG) // Take the stuffed sprite of the same size
								size = G.size
								used_icon_location = icon_stuffed_location
							if(FULLNESS_LEVEL_BEEG to FULLNESS_LEVEL_NOMOREPLZ) // Take the stuffed sprite of size + 1
								size = G.size + 1
								used_icon_location = icon_stuffed_location
							if(FULLNESS_LEVEL_NOMOREPLZ to INFINITY)// Take the stuffed sprite of size + 2
								size = G.size + 2
								used_icon_location = icon_stuffed_location
								
```
I'd recommend adding spacing so that you can break up long sections of code. It'll make it easier for people to read and understand.